core/src/main/java/com/dtp/core/spring/DtpLifecycleSupport.java

```diff
@@ -22,7 +23,8 @@
  * @since 1.0.3
  **/
 @Slf4j
-public abstract class DtpLifecycleSupport extends ThreadPoolExecutor implements InitializingBean, DisposableBean {
+public abstract class DtpLifecycleSupport extends ThreadPoolExecutor
+        implements InitializingBean, DisposableBean, ExecutorAdapter<ThreadPoolExecutor> {
```

Comment:
I think DtpExecutor can be used to implement the ExecutorAdapter interface, so that DtpLifecycleSupport only deals with lifecycle-related semantics.

Reply:
> I think DtpExecutor can be used to implement the ExecutorAdapter interface, so that DtpLifecycleSupport only deals with lifecycle-related semantics.

agree and solved